### DISCUSSION TITLE
PDF Translation

### DISCUSSION THREAD

**Discussion by CyberSardinha** [Stirling-Tools/Stirling-PDF#262] [Ideas]
It would be great if we could do PDF Translation as well...

it could use deep-translator or Argos Translate

What do you guys think?

**Comment by Frooodle** [comment]
This would also need a find replace feature added prior to this 

I think translation might take up a good amount of space, would be good if it uses packs which can be added to volume or if it points to another server
I wouldn't want to add directly to spdf package
